kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/src/test/java/org/kie/workbench/common/dmn/showcase/client/selenium/DMNDesignerKogitoSeleniumIT.java

```diff
@@ -86,12 +90,31 @@ public void teardown() {
         }
     }
 
+    @Test
+    public void testNewDiagram() throws Exception {
+        setContent("");
+
+        final String xml = getContent();
+        assertThat(xml).isNotBlank();
+
+        System.out.println(xml);
```

Comment:
I prefer compare with some specific text than just printing and watching jenkins logs :)

Reply:
Oh sure.. this test would need refactoring to check the returned XML matches a "new diagram" template.. let me do that now (in a new commit to follow).